marketplace_builder/views/pages/tutorials/forms/form-server-side-processing.liquid

```diff
@@ -0,0 +1,68 @@
+{% endcapture %}
+{% include 'alert/important', content: content %}
+
+
+## Find FormConfiguration based on `form_configuration_name` or `form_configuration_name` parameter
+
+The first step is to identify FormConfiguration to be used to process user's request. In order to do it, the server checks for parameter called `form_configuration_name` or `form_configuration_id`, which is automatically added to the page when form is rendered using {% raw %}`{% form %}`{% endraw %} tag as described in [FormTag](https://documentation.platform-os.com/tutorials/forms/form-rendering#form-tag). If the value is missing or invalid, 404 error is rendered, otherwise we move on.
+
+## Check Authorization Policies
+
+The next step is authorization using [Authorization Policies](https://documentation.platform-os.com/tutorials/authorization-policy/authorization-policy). Those are processed in the order in which we were defined - from top to bottom. If the authorization fails, the server stops processing the form and handles authorization policy failure(usually redirect or rendering 403 error).
+
+## ResourceOwner authorization
+
+Once all authorization policies pass, the next check is authorization ResourceOwner property. If fails, the server returns 404 error.
+
+## Decorating user input with `default_payload` json
+
+The next step is to extend/overwrite user's input with `default_payload`. This is the place when liquid is evaluated server side. The most useful variables here used to build default_payload are `context.params` and `form` variables. If evaluated liquid is not valid json, the server raises 500 error and automatically creates relevant [Log](https://documentation.platform-os.com/tutorials/qa/creating-marketplace-kit-logs).
+
+## Validating user input
+
+The result of merging user's input with default_payload is then run against validation rules defined for each field. If at least one validation fails, the server renders response based on the format. For html the errors are added to `form_builder` variable inside the relevant field and server renders [Page](https://documentation.platform-os.com/tutorials/pages/pages) based on `page_id` parameter passed to the server. Additionally, the flash_alert is evaluated and set if provided in the configuration. It's then accessible using `context` variable. For JSON, a json object is returned with populated `errors` key. The status code is 200 in both scenarios.
+
+## Calculating changes
+
+If validation pass, the changeset between existing state of the resource and user input is calculated, and accessible from now on via `changes` variable.
+
+## Persisting changes
+
+The new state of the resource is persisted in the database. In case of rare scenario of internal server error, the changes are rollbacked and server renders 500 error.
+
+## Processing Callbacks
+
+The next step is to process the callback synchronously.
```

Comment:
It looks like this article could use an ordered list or at least numbers before each step